dcpy/lifecycle/ingest/templates/dpr_capitalprojects.yml

```diff
@@ -5,20 +5,18 @@ source:
   url: https://www.nycgovparks.org/bigapps/DPR_CapitalProjectTracker_001.json
 file_format:
   type: json
-  normalize:
-    record_path:
-    - Locations
-    - Location
-    meta:
-    - TrackerID
-    - FMSID
-    - Title
-    - TotalFunding
+  json_read_meta:
+    json_read_fn: normalize
+    json_read_kwargs:
+      {
```

Comment:
I was going to say this could just be yml, but there's something nice about making it look a little more like actual function args. Definitely at least like one-lining the arrays, undecided on the brackets